variables.tf

```diff
@@ -750,6 +750,17 @@ variable "runner_worker_docker_machine_autoscaling_options" {
   default = []
 }
 
+variable "runner_worker_graceful_terminate" {
+  description = "Gracefully terminate Runner Worker, by waiting a set amount of time for running jobs to finish before termination."
+  type = object({
+    enabled      = optional(bool, false)
+    timeout      = optional(number, 1800)
```

Comment:
docs: this should be the maximum duration of the longest job running on the workers/executors. 1800s seems good for me, but a documentation should be added. Setting this value too low will result in not gracefully shutdown the Runner.

Reply:
Updated docs for `timeout` in README.